workers/unity/Assets/Fps/Scripts/SetupLogic/ClientWorkerConnector.cs

```diff
@@ -109,6 +112,12 @@ protected override void HandleWorkerConnectionEstablished()
             base.HandleWorkerConnectionEstablished();
         }
 
+        private void RemovingAuthoritativePlayer()
+        {
+            Debug.LogError($"Removing authoritative player.");
```

Comment:
This message doesn't feel like an error. Might need some clarification in the message as to why this is bad.
"Player entity was removed while connected, disconnecting" ?